src/JSInterop/Microsoft.JSInterop/test/JSRuntimeTest.cs

```diff
@@ -297,19 +297,9 @@ public void SerializesDotNetObjectWrappersInKnownFormat()
         public void CanSanitizeDotNetInteropExceptions()
         {
             // Arrange
-            var expectedMessage = "An error ocurred while invoking '[Assembly]::Method'. Swapping to 'Development' environment will " +
-                "display more detailed information about the error that occurred.";
-
-            string GetMessage(DotNetInvocationInfo info) => $"An error ocurred while invoking '[{info.AssemblyName}]::{info.MethodIdentifier}'. Swapping to 'Development' environment will " +
-                "display more detailed information about the error that occurred.";
```

Comment:
This was a bit odd before because it looked as if it was testing product code but wasn't. I've made the assertions a bit more direct now.